plan/plan.go

```diff
@@ -175,3 +175,27 @@ func shouldUpdateTTL(desired, current *endpoint.Endpoint) bool {
 	}
 	return desired.RecordTTL != current.RecordTTL
 }
+
+// filterRecordsForPlan removes records that are not relevant to the planner.
+// Currently this just removes TXT records to prevent them from being
```

Comment:
"currently this just removes TXT records" --- I think the comment is now wrong as all records except `A` and `CNAME` are filtered out...

Reply:
I say currently because TXT, CNAME and A are the only record types that ExternalDNS looks at, as far as I can tell. Should I reword that?